service/src/main/java/com/fecklessweasel/service/UniversityServlet.java

```diff
@@ -48,6 +49,7 @@ public Integer run(Connection connection)
 			// Redirect to homepage.
 			//response.sendRedirect("/");
 		request.setAttribute("longName", longName);
-		request.getRequestDispatcher("/classroom.jsp").forward(request, response);
-	}
+		request.setAttribute("state", state);
```

Comment:
I'm thinking that this is a dangerous precedent we're setting here. This is saving state between pages, meaning that the UniversityPage URL we are redirecting to will only work this one time. 

If the user refreshes their browser, this can potentially break. Also, multiple users will cause things to break.

We should instead use query params in the URL. e.g.: /UniversityPage.jsp?uid=34535

The UniversityPage.jsp file can then load the university by checking the ID and querying the database for it.